desktop/apps/categories3/helpers/alphabetizeGenes.js

```diff
@@ -0,0 +1,3 @@
+import _ from 'underscore'
+
+export default genes => _.sortBy(genes, gene => gene.name)
```

Comment:
👍 If this section grows and you start to need a lot of utils and they can be grouped, I would go back to the `utils/index.js` pattern, but if not this feels much cleaner. Can also fuzzy find right to it if you need to make updates.

Reply:
Yeah I suspect I'll be doing that in the next PR 😸